### DISCUSSION TITLE
Prototype of Implicit Locatives Reference Page for Thursday's meeting

### DISCUSSION THREAD

**Discussion by EricZinda** [delph-in/docs#102] [Team Posts]
I don't know if I need to "@" people to get noticed in Github, so: @arademaker, @fcbond, @danflick, @olzama, @lmorgadodacosta:
 
I finished a [prototype of the reference page for Implicit Locatives](https://github.com/delph-in/docs/wiki/ERDW_ImplicitLocatives). The original is [here](https://github.com/delph-in/docs/wiki/ErgSemantics_ImplicitLocatives).  The prototype attempts to describe the concept in a way that the target audience we discussed can understand. I also tried to make it work as it does now by keeping all the linguist-specific info in there.

The questions I have are:
- Overall thoughts and comments?
- Do you think it works for the target audience?
- Do you think it succeeds in targeting both audiences?  I.e. if we deleted the original, are we good with that?

We can discuss in this thread and at the meeting.

More details if you want to know what changed:
- Start with the description since the examples don’t make much sense until you understand the idea
- Get rid of linguistic jargon in the description
- Move a complicated case from the description to “Reflections” section (“Another characteristic construction where a spatio-temporal modifier…”)
- Add explanations to the examples (i.e. “Where is the stick? -> no relationship other than the stick "is" somewhere”). For me, it wasn’t obvious why this was “nonspecific” even after reading the overview for loc_nonsp, so I tried to spell it out
- Add “Counter Examples” to clarify when it *isn’t* non_specific
- Spell out the predicates that get generated instead of just relying on the fingerprint of “loc_nonsp[ARG1 i, ARG2 x]” since a) that isn’t what shows up in the MRS, b) that language is kind of complicated
- Give examples of simple MRS for each so you can see it in action
- Spell out what the args do for the predicates that get generated, in non-lingustic terms (when possible. “Event” needs to be understood. Ditto “Entity”

I left everything else in there that was there before, but at the bottom. It seems like all audiences would benefit from the rewrite of the top, but having all the background at the bottom would be helpful for linguists, grad students, etc. And: I don’t think it scares anyone because they’ll just learn to stop reading those parts.

**Comment by arademaker** [comment]
> ERG (1212) doesn't parse Browne arrived the tobacco garden day. Are noun-noun compounds not the sort of thing that can take a word of the day type and make it into a phrase that can function as a loc_nonsp modifier?

why this is still relevant if ERG last version is 2020!?

**Comment by arademaker** [comment]
I fixed the link in your post above, to the original version.

**Comment by arademaker** [comment]
I have a hard time trying to find the changes you made in the original text, so creating a copy of the page is not a good way to go. Considering that @emilymbender may want to review changes, I believe we need to find a better approach. Maybe google docs? Maybe the docs repository itself?

**Comment by arademaker** [comment]
> locates the stick in a place on the location

This is a typo?

**Comment by arademaker** [comment]
> The dog barked on Tuesday./The dog barked every Tuesday.

I failed to see the parallelism of these two.

**Comment by arademaker** [comment]
The MRS can be made simpler when I don't need to talk about the variable properties, see [here](properties).

```
[ TOP: h0
INDEX: e2
RELS: <  
 [ which_q LBL: h6 ARG0: x4 RSTR: h7 BODY: h8 ]
 [ place_n LBL: h5 ARG0: x4  ]
 [ _the_q LBL: h9 ARG0: x3 RSTR: h10 BODY: h11 ]
 [ _stick_n_1 LBL: h12 ARG0: x3  ]
 [ loc_nonsp LBL: h1 ARG0: e2 ARG1: x3 ARG2: x4 ] >
 HCONS: < h0 qeq h1 h7 qeq h5 h10 qeq h12 > ]
```

**Comment by arademaker** [comment]
In https://github.com/delph-in/docs/wiki/ERDW_ImplicitLocatives#mrs-expression, does it make sense to consider two forms only because the argument `ARG1` can be event or individual? Note that the [fingerprint](https://github.com/delph-in/docs/wiki/ERDW_ImplicitLocatives#ers-fingerprints) already says its type is `i` is already a generalization of `e` and `x`, see [here](https://github.com/delph-in/docs/wiki/ErgSemantics_Basics#variable-types).

**Comment by arademaker** [comment]
the name `loc_nonsp` evoke this interpretation of `nonspecific`. The section `motivating examples` is confusing... it says

> This analysis involves a certain amount of ‘decomposition’ in the semantics; the alternative would be to have predicate symbols such as today_n or here_n which directly take events as the value of their ARG1. 

but [the stick is here](http://delph-in.github.io/delphin-viz/demo/#input=The%20stick%20is%20here&count=5&grammar=erg2018-uw&mrs=true) has the predicate `_here_a_1` that has ARG1 with an entity.

**Comment by arademaker** [comment]
Examples can be taken from the profiles under http://svn.delph-in.net/erg/trunk/tsdb/gold/. I think the [LTDB](http://compling.hss.ntu.edu.sg/ltdb/cgi/ERG_1214//ltypes.cgi) does not have interface to query for predicates, I was able to find examples using grep in my local copy of the ERG SVN tree. But this would be the ideal use of the https://github.com/own-pt/wsi, I need to clean my student code to make it useful for others.

<img width="1807" alt="image" src="https://user-images.githubusercontent.com/86403/178755634-91c70c86-eb6b-4b0a-8e38-0ed854dcc488.png">

**Comment by EricZinda** [comment]
> ERG (1212) doesn't parse Browne arrived the tobacco garden day. Are noun-noun compounds not the sort of thing that can take a word of the day type and make it into a phrase that can function as a loc_nonsp modifier?
> 
> why this is still relevant if ERG last version is 2020!?

I didn't have the background to know whether this should go or stay so I left it to not lose information.  This is definitely a place for the "technical editor" pass to weigh in.

**Comment by EricZinda** [comment]
> I fixed the link in your post above, to the original version.

The magic of wiki. Thanks!

**Comment by EricZinda** [comment]
> I have a hard time trying to find the changes you made in the original text, so creating a copy of the page is not a good way to go. Considering that @emilymbender may want to review changes, I believe we need to find a better approach. Maybe google docs? Maybe the docs repository itself?

I suspect this is a question to answer when we've settled on a prototype we like since diff'ing is much less useful the more restructuring goes on.  If there are smaller changes it works. We should also keep in mind that there are existing pages and there are new pages.  We may need to go through some pain for existing pages by, for example, viewing them side by side, or reading them one after the other.  New pages probably will work fine with a diffing approach

**Comment by EricZinda** [comment]
> locates the stick in a place on the location
> 
> This is a typo?

No, but obviously confusing. I was trying to say it places the stick somewhere specific in the location as opposed to just "at" the location. Happy for better suggestions!

**Comment by EricZinda** [comment]
> The dog barked on Tuesday./The dog barked every Tuesday.
> 
> I failed to see the parallelism of these two.

This was just left from the original.  I think this is a question for @danflick

**Comment by EricZinda** [comment]
> The MRS can be made simpler when I don't need to talk about the variable properties, see [here](properties).
> 
> ```
> [ TOP: h0
> INDEX: e2
> RELS: <  
>  [ which_q LBL: h6 ARG0: x4 RSTR: h7 BODY: h8 ]
>  [ place_n LBL: h5 ARG0: x4  ]
>  [ _the_q LBL: h9 ARG0: x3 RSTR: h10 BODY: h11 ]
>  [ _stick_n_1 LBL: h12 ARG0: x3  ]
>  [ loc_nonsp LBL: h1 ARG0: e2 ARG1: x3 ARG2: x4 ] >
>  HCONS: < h0 qeq h1 h7 qeq h5 h10 qeq h12 > ]
> ```

(that link didn't work for me). Good discussion for "formats and standards" for the docs. Do we always use full MRS? Simple MRS? Is it OK to use fragments? etc. Worth at least a short discussion to pick our main approach, there will always be exceptions, of course.

I like the idea of leaving out the properties, BTW, because it is way more readable...

**Comment by EricZinda** [comment]
> In https://github.com/delph-in/docs/wiki/ERDW_ImplicitLocatives#mrs-expression, does it make sense to consider two forms only because the argument ARG1 can be event or individual? Note that the fingerprint already says its type is i is already a generalization of e and x, see here.

I wanted to spell out the alternatives so I could show examples (along with their MRS) and show a description of what the actual arguments mean for each alternative. In my own use, it is often "obvious" what the arg means in a particular usage, but then I have to learn the hard way that what I thought it meant was wrong when I hit a different usage of the same predicate.  I feel like we shouldn't make people have to guess this stuff.

**Comment by EricZinda** [comment]
> the name loc_nonsp evoke this interpretation of nonspecific. The section motivating examples is confusing... it says
> 
> This analysis involves a certain amount of ‘decomposition’ in the semantics; the alternative would be to have predicate symbols such as today_n or here_n which directly take events as the value of their ARG1.
> 
> but the stick is here has the predicate _here_a_1 that has ARG1 with an entity.

Good question for @danflick .  Again, I just left this section as it was

**Comment by EricZinda** [comment]
> Examples can be taken from the profiles under http://svn.delph-in.net/erg/trunk/tsdb/gold/. I think the LTDB does not have interface to query for predicates, I was able to find examples using grep in my local copy of the ERG SVN tree. But this would be the ideal use of the https://github.com/own-pt/wsi, I need to clean my student code to make it useful for others.

This will be very helpful! Let's discuss that tool a bit more in the meeting. I'm not familiar with it.

**Comment by arademaker** [comment]
> I feel like we shouldn't make people have to guess this stuff.

But it would not be guessed, the opposite. The section https://github.com/delph-in/docs/wiki/ERDW_ImplicitLocatives#ers-fingerprints says ARG1 is type `i`, this means it can be `i`, `x` or `e`. See https://github.com/delph-in/docs/wiki/ErgSemantics_Interface and the file `etc/erg.smi` in the ERG SVN repository.

**Comment by EricZinda** [comment]
> But it would not be guessed, the opposite.

I don't mean the type, necessarily, I mean the *role* the arguments are playing.   The fact that an argument is an instance is great, but for a particular predicate that has X, X, X, which is doing what? in compound(x, x) which side means what? etc.

**Comment by fcbond** [comment]
We should definitely describe this in the documentation for the lexical
entries.

**Comment by arademaker** [comment]
Maybe following some ideas of the roleset description of propbank, e.g. https://verbs.colorado.edu/propbank/framesets-english-aliases/drive.html. But we have many `drive` lexicon entries in ERG, and some predications like `_drive_v_1_rel`, can we fix an interpretation of the arguments for each predicate?

**Comment by EricZinda** [comment]
Exactly! Don't know if we can be as specific as they are, but it would be so useful
